### PR TITLE
[MOSIP-31911] Automated Creation of readuser for postgres,minio and k…

### PR DESCRIPTION
**Pull request by Mohanraj209**
…eycloak service.

### CODE REVIEW

File: charts/keycloak-init/values.yaml [hunk 1]
```diff
@@ -657,7 +657,25 @@ keycloak:
             - SUBSCRIBE_OIDC_CLIENT_CREATED_GENERAL
             - SUBSCRIBE_OIDC_CLIENT_UPDATED_GENERAL
             - PUBLISH_AUTHENTICATION_ERRORS_GENERAL
-      users: []
+      users:
```
Comment (ckm007) [inline charts/keycloak-init/values.yaml:660]: @Mohanraj209 We dont want want this read user as part of default installation.
this should be executed on need basis hence can be moved to the same utils.

**Review by ckm007 — COMMENTED** [review]
Instead of having two different charts for miniop and postgres for readuser access. We need one chart which creates both the jobs and their configurations are provided using same values.
Along with this in mosip infra will  run keycloak-init with only user related values.

File: charts/keycloak-init/values.yaml (hunk 1 above)
Comment (Mohanraj209) [reply charts/keycloak-init/values.yaml:660]: sure sir

**Comment by Mohanraj209** [comment]
> Instead of having two different charts for miniop and postgres for readuser access. We need one chart which creates both the jobs and their configurations are provided using same values. Along with this in mosip infra will run keycloak-init with only user related values.

done sir

File: charts/readuser-util/values.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+postgres-readuser-util:
+  enabled: true
+  # Values for readuser creation.
+  # "action" key specifies whether you want to "create" or "delete" readuser.
+  # "username" key specifies the username of readuser creating.
+  # "password" key specifies the password for readuser.
+  # "dbport" key specifies the DB port number.
+  # "dbhost" key specifies the DB host.
+  user:
+    action: "create"
```
Comment (ckm007) [inline charts/readuser-util/values.yaml:14]: add possible actions in comments

Comment (Mohanraj209) [reply charts/readuser-util/values.yaml:14]: Updated

**Review by ckm007 — APPROVED** [review]

**Pull request merged**